### PR TITLE
[18187] CLI: Reliability QoS implementation

### PR DESCRIPTION
**Pull request by JLBuenoLopez-eProsima**

### CODE REVIEW

File: cli/src/cpp/usages.hpp [hunk 1]
```diff
@@ -457,7 +457,7 @@ static const char PARTICIPANT_NAME_USAGE[] =
 
     The <profile_name> element is MANDATORY and it can be any string (whitespaces are supported if quoted).
 
-    The <participant_name> value is MANDATORY and it can be any string (whitespaces are supported if quoted).
+    SET command REQUIRES a <participant_name> value. It can be any string (whitespaces are supported if quoted).
```
Comment (JesusPoderoso) [inline cli/src/cpp/usages.hpp:460]: I think it should be mandatory as it was

File: cli/test/README.md
Comment (JesusPoderoso) [inline cli/test/README.md:1]: Shall the footer be modified also?

`[^3]: **BUG**: Access to file before previous changes have been written although Xerces has correctly returned from write operation.`

File: cli/src/cpp/usages.hpp (hunk 1 above)
Comment (JLBuenoLopez-eProsima) [reply cli/src/cpp/usages.hpp:460]: The `participant_name` is the argument passed to the CLI, and it is only required for `SET` command. `CLEAR` or `PRINT` does not require the name, only the `participant_profile` which is still mandatory.

Comment (JesusPoderoso) [reply cli/src/cpp/usages.hpp:460]: I confused it with the `<profile_name>`, you are right

**Review by JesusPoderoso — APPROVED** [review]
LGTM

**Pull request merged**